client_firmware/client_firmware.ino

```diff
@@ -1,17 +1,18 @@
 #include <EEPROM.h>
 #include <EtherCard.h>
-#include <IPAddress.h>
 #include <avr/wdt.h>
 
 #define CSPIN 10
 // NOTE: All of these timers are in Milliseconds!
-// Ping our target every 2 seconds
-#define PINGER_INTERVAL 2000
+// Ping our target every 60 seconds
+#define PINGER_INTERVAL 60000
```

Comment:
I used to have multicast and broadcast reception on in my custom packet loop, but long term it was unstable.
Now, at the time it could have been any number of things (ram usage?), so maybe it might be a good time to experiment with that back on. I agree that picking up on broadcasts would be helpful.